src/components/molecules/NavBar/NavBar.scss

```diff
@@ -270,4 +272,30 @@ $border-radius: 28px;
   &__profile-popover {
     height: 100%;
   }
+
+  &__photobooth {
+    &-button {
```

Comment:
```suggestion
    &--button {
```

Reply:
Title isn't modifier here. The & is making it a bit more complex to notice that it's actually part of a more complex name.

`NavBar__photobooth-button`

My suggestion would be to just unroll this case as `&__photobooth-button` without the double `&` concatenation.

Reply:
fixed b230401